php/php.dbgp/src/org/netbeans/modules/php/dbgp/breakpoints/BreakpointsReader.java

```diff
@@ -34,14 +34,16 @@ public class BreakpointsReader implements Properties.Reader {
 
     private static final String ENABED = "enabled"; // NOI18N
```

Comment:
Although it is not your change, I would change the name.
```suggestion
    private static final String ENABELED = "enabled"; // NOI18N
```

Reply:
Yes, please. Thanks.